### ISSUE TITLE
Farm Statistic Suite / Farm Logbook

### ISSUE THREAD

**Issue by Vestrelys** [StardewModders/mod-ideas#492]
Thanks to @Pathoschild for let me share this idea.
For Modders i have favor to ask though since i just have idea not skill to code,
 i've been wondering it would be great if there's mod for statistic or detailed log like about how much every goods we produced, how much goods we gathered, how much we stored, how much we sell/shipping, Produced Value, Stored Value, Sold Value, Profit Value in detailed table in-game.

Below i just make roughly layout using notepad >,< I hope you guys get the idea.

![Request Mod_Farm Statistic Suite](https://user-images.githubusercontent.com/67378861/85587485-03851d00-b66c-11ea-8cd7-af7fb8300af1.jpg)

This mod purpose is player can get information about productivity and management farm(not only profit) so they know and can plan for next week,season,year. Make the game more interesting for some player.

**Features** 
##This mod in-game window the information can be viewed with 4 different Period as per week/ per Season/ per Year/ per Lifetime.

##Information displayed by category each tab/page, here the list:
-Crops ( 1st Tab/page)
-Fruit Tree ( 2nd Tab/page)
-Gathered/Foraged ( 3rd Tab/Page)
-Artisan Goods ( 4th Tab/Page)
-Mined/Geodes Processed ( 5th Tab/Page)(Optional)
-Summary ( Last Tab/Page)
(the content summary page like below)
>>Crops : Total Harvested  || Total Harvested Value ||Total Stored || Total Stored Value || Total Shipped/Sold || Total Shipped/Sold Value || Total Profit ||

>>Fruits : Total Harvested  || Total Harvested Value ||Total Stored || Total Stored Value || Total Shipped/Sold || Total Shipped/Sold Value || Total Profit ||

>>Artisan Goods : Total Made/Processed  || Total Made/Processed Value || Total Stored || Total Stored Value || Total Shipped/Sold || Total Shipped/Sold Value || Total Profit ||

>>Mined (if made) : Total Minerals Found || Total Minerals Value || Total Stored || Total Stored Value || Total Shipped/Sold || Total Shipped/Sold Value || Total Profit ||

>>Geodes Processed (if made) : Total Geodes Processed || Total Geodes Processed Value || Total Stored || Total Stored Value || Total Shipped/Sold || Total Shipped/Sold Value ||

>>Upgrades : Total FarmHouse Upgrades Spent || Total Tools Upgrades Spent || Total Farm Upgrades Spent (Coop,Barn,etc)  || Total Upgrades Spent

>>(Farm Name) Farm Asset : Total Farm Produced Value || Total Farm Stored Value || Total Farm Shipped/Sold Value || Total Farm/Player Earnings || Total Upgrades Spent || Total Profit

**Note for modder :** 
- Mod can be made Independent window in-game(require new hotkey to open window) or new tab integrated on Menu in-game(hotkeyless)
- Basically the point idea of this mod is can provide player to see track record their game and analyze/balancing their financial for next season or year from information productivity in this mod.
- For some player info provided in this statistic can become achievement or target to achieve for self-achiever pleasure.
- i believe this additional UI mod can also tagged as Quality of Life
- i just can suggest this idea and rough layout since idk about code or make mod, **all credits goes to all modder who can make this mod and publishing in nexusmod**

**Comment by tiakall** [comment]
I believe the Farm Computer and the Qi casino stat board, introduced in 1.5, covers a lot of this!
